content/legal/terms-and-conditions.md

```diff
@@ -0,0 +1,259 @@
++++
+title = "Matrix.org Homeserver Terms and Conditions"
++++
+
+Please read this document carefully before accessing or using this service!
+
+# 1. Introduction
+
+## 1.1 English, Not Legalese
+
+Most Terms of Use and Privacy Policy documents are unreadable. They are written by lawyers and for lawyers, and in our opinion are not very effective.
+
+We decided to use plain English as much as possible, to make our terms as clear as possible. Some sections still have room for improvement - we plan to tackle these over time.
```

Comment:
"as much as possible" doesn't seem to be a whole lot given the many "Note on Plain English" sections below. I'm wondering whether we should really have this paragraph – is sbd. actively working on making things easier to understand?

Reply:
This is legal team territory, I'd rather port it as is and fix it with them eventually :)